apiClient/client.ts

```diff
@@ -72,15 +72,21 @@ export async function updateUser(id: number, partial: PartialUser) {
   return await res.json()
 }
 
-export async function listLeaderboard({
+export async function listUsers({
   search,
   country_code: countryCode,
   event_type: eventType,
+  limit,
+  after,
+  before,
 }: {
   search?: string
   country_code?: string
   event_type?: string
-} = {}): Promise<ListLeaderboardResponse | ApiError> {
+  limit?: number
+  after?: number
+  before?: number
+}): Promise<ListLeaderboardResponse | ApiError> {
```

Comment:
It'd be nice to rename the response type to `ListUsersResponse` if we're going to rename the function (I agree renaming the function makes sense)